typescript/sdk/src/gas/types.ts

```diff
@@ -7,16 +7,16 @@ import type { CheckerViolation, OwnableConfig } from '../deploy/types';
 import { ChainMap } from '../types';
 
 import { IgpFactories } from './contracts';
-
-export enum GasOracleContractType {
-  StorageGasOracle = 'StorageGasOracle',
-}
+import { DestinationOracleConfig, GasOracleContractType } from './oracle/types';
 
 export type IgpConfig = OwnableConfig<keyof IgpFactories> & {
   beneficiary: Address;
-  gasOracleType: ChainMap<GasOracleContractType>;
   oracleKey: Address;
   overhead: ChainMap<number>;
+  // TODO: require this
```

Comment:
why the TODO

Reply:
at present, without configuring this key, the IGP that gets deployed is not useful